src/js/media-seek-backward-button.js

```diff
@@ -69,13 +53,47 @@ class MediaSeekBackwardButton extends MediaChromeButton {
       if (newValue == undefined) {
         this.setAttribute(Attributes.SEEK_OFFSET, DEFAULT_SEEK_OFFSET);
       }
-      updateSeekIconValue(this);
+      updateSeekIconValue(this, 'backward');
       updateAriaLabel(this);
     }
 
     super.attributeChangedCallback(attrName, _oldValue, newValue);
   }
 
+  // Own props
+
+  /**
+   * @type {number} Seek amount in seconds
+   */
+  get seekOffset() {
+    return +this.getAttribute(Attributes.SEEK_OFFSET);
+  }
+
+  set seekOffset(value) {
+    // avoid triggeting a set if no change
+    if (value === this.seekOffset) return;
+    this.setAttribute(Attributes.SEEK_OFFSET, value.toString(10));
```

Comment:
same here with null / undefined and string numbers

Reply:
These are easy to account for but more a matter of how we would like the API to work e.g.

- Explicit, use the the type we define or expect an error
- Implicit, we coerce many value types into the type we want

I have a slight preference for the former but I'm not that passionate about it 😅. Is being more implicit in what we accept closer to how others would expect it to work? if the type is number is it expected that '50' works? or should the type be `number | string` to explicitly state that we support it.

Reply:
We don't have a consistent stance on this, but we probably should. While I normally have your leanings, @AdamJaggard, here are my thoughts specifically for Media Chrome:

1. I think we can have fairly strict TypeScript definitions, such that, when folks **_are_** using it, the errors will inform them on what kind of values are expected. 
2. I also think we want to be a bit more permissive "under the hood", in the direction that @luwes is suggesting, in what is allowed in JS. This will keep our implementation closer to the way native `HTMLElement`s actually behave, and keeping Media Chrome consistent with native elements is a design principle.

This is effectively the relationship between the official TypeScript definitions of elements in lib.dom.d.ts vs. the actual implementation/behavior of the elements. By way of example, the definition of `HTMLMediaElement` in lib.dom.d.ts includes:

```ts
/** Gets or sets the volume level for audio portions of the media element. */
volume: number;
```
And here's how an `HTMLVideoElement` actually behaves in the browser JS runtime:
```js
videoEl.volume = '0.5';
videoEl.volume; // 0.5
videoEl.volume = true;
videoEl.volume; // 1
videoEl.volume = 'foo'; // caught TypeError: Failed to set the 'volume' property on 'HTMLMediaElement': The provided double value is non-finite.
```

Given that we also cast values _**back**_ to strings (as attributes), if we wanted the widest support, we'd need to make sure we cast to a number before applying the value to `setAttribute()`.

There are other reference points that could lead to slightly different conclusions, though I think those would be the wrong direction. For example, an `HTMLInputElement` of `type="range"` has its `min`/`max/`step` properties typed & casted as a `string`, which leads to much worse ergonomics imo.

This isn't saying we _**have**_ to do things this way, but I do think it best meets the various background considerations specifically applicable to Media Chrome. If there's lingering disagreement here, I think we should defer to Adam to allow him to continue making fast forward progress and take this up out of band (ideally soon & async, but we can wait until our office hours if needed).

Reply:
Having stricter type defs while being more permissive in the js sounds good to me.

How did we want to handle falsey values? Remove the attribute? Or only do it for undefined/null? Interestingly, setting volume to `undefined` throws an error, but setting it to `null` sets it to 0.

Reply:
@gkatsev that's because:
```js
+null // 0
+undefined // NaN
```

Reply:
Have updated these to handle null and undefined together. String numbers already work fine as string.toString() already works as expected.